get-started/02_our_app.md

```diff
@@ -52,10 +80,11 @@ see a few flaws in the Dockerfile below. But, don't worry. We'll go over them.
    CMD ["node", "src/index.js"]
    EXPOSE 3000
    ```
+  > **Tip**
```

Comment:
Note?

Reply:
Thanks @jerae-duffin. Updated in the following commit.